testing/taxonomies/MessageShape.ttl

```diff
@@ -35,7 +35,35 @@ shapes:DescriptionRequestMessage
 	] .
 
 
+shapes:RequestMessageShape
+	a sh:NodeShape ;
+	sh:targetClass ids:RequestMessage ;
+
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:callbackURL ;
+		sh:datatype xsd:anyURI ;
+		sh:minCount 1 ;
```

Comment:
According to the shape, a callback URL is required. Does this imply that synchronous messages are not supported anymore? Or would an empty string URL suggest a synchronous message?

Reply:
sh:minCount 0;
sh.maxCount 1; 

Should solve the problem here. That leads at last to the following conclusion:
If callbackURL is filled --> asynchronous communication if it is empty --> synchronous

Reply:
@changqin26 could you please review and implement this change? Thank you!

Reply:
Thanks for the hint, already corrected it.